### PR TITLE
Added files for review

### PR DESCRIPTION
**Pull request by surbhig12**
Please review the files.

### CODE REVIEW

File: intro_tizen_net.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Introduction to Tizen .NET Application
+
+Tizen .NET is an exiting new way to develop applications for the Tizen operating system, running on 50 million Samsung devices, including TVs, wearables, mobile phones, and many other IoT devices around the world.
```
Comment (Reni-90) [inline intro_tizen_net.md]: Spelling correction:

Please change "exiting" to "exciting"

File: intro_tizen_net.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+The existing Tizen frameworks are either C-based with no advantages of a managed runtime, or HTML5-based with fewer features and lower performance than the C-based solution. With Tizen .NET, you can use the C# programming language, the Common Language Infrastructure standards, and benefits from a managed runtime for faster application development and code execution, which is efficient and secure
```
Comment (Reni-90) [inline intro_tizen_net.md]: Please add a period at the end of the sentence.

.... efficient and secure.

File: intro_tizen_net.md (hunk 1 above)
Comment (surbhig12) [reply intro_tizen_net.md]: Done

File: intro_tizen_net.md (hunk 2 above)
Comment (surbhig12) [reply intro_tizen_net.md]: Done

**Comment by annie-samsung** [comment]
Merging this file.

**Pull request merged**